Source/cielim/Private/ProtobufActor.cpp

```diff
@@ -1,35 +1,205 @@
 // Fill out your copyright notice in the Description page of Project Settings.
 
 #include "ProtobufActor.h"
-#include <iostream>
-#include <fstream>
-#include <sstream>
-
+#include "KinematicsUtilities.h"
+#include "Engine/World.h"
 
 // Sets default values
 AProtobufActor::AProtobufActor()
 {
-    // Set this actor to call Tick() every frame.  You can turn this off to improve performance if you don't need it.
-    PrimaryActorTick.bCanEverTick = true;
-
+ 	// Set this actor to call Tick() every frame.  You can turn this off to improve performance if you don't need it.
+	PrimaryActorTick.bCanEverTick = true;
```

Comment:
:small_red_triangle_down: There is some odd indentation throughout the files in this PR. Maybe IDE settings?

Reply:
This one was weird because I did not see the indentation on my editor, Ill check if this is still an issue